### ISSUE TITLE
Figure out what do with lcamtuf's email address

### ISSUE THREAD

**Issue by jonathanmetzman** [google/AFL#32]
Emailing lcamtuf@google.com is no longer the correct way to raise issues about AFL, in part because I don't think this email email address is active anymore (see #30) for example.
We should probably replace references to it with the afl-users mailing list or this tracker.

**Comment by Dor1s** [comment]
Yeah, maybe just remove the email (but can leave the name). Some other pages e.g. README are pointing to the mailing list already, that might be enough.
